monitoring/rid_qualifier/flight_data_generator.py

```diff
@@ -0,0 +1,481 @@
+from shapely.geometry import Point, Polygon, LineString
+import shapely.geometry
+from pyproj import Geod, Transformer, Proj
+import json
+from pathlib import Path
+from typing import List, NamedTuple, Any
+import arrow
+import datetime
+from datetime import datetime, timedelta
+
+
+class QueryBoundingBox(NamedTuple):
+    ''' This is the object that stores details of query bounding box '''
+
+    name: str
+    shape: Polygon
+    timestamp_before: timedelta
```

Comment:
Timestamps should generally be of type `datetime` (in package `datetime`) rather than `timedelta`, and indeed line 163 populates these fields with `datetime`s and line 373 assumes they are `datetime`s.

Reply:
Correct! Got it! I have now made them date time.